features/device-config/monitor/src/main/java/org/opennms/features/deviceconfig/monitors/DeviceConfigMonitor.java

```diff
@@ -129,7 +132,13 @@ public PollStatus poll(MonitoredService svc, Map<String, Object> parameters) {
         final Date nextRun = getNextRunDate(cronSchedule, lastRun);
 
         if (!triggeredPoll && !nextRun.before(new Date())) {
-            return PollStatus.unknown("Skipping. Next retrieval scheduled for " + nextRun);
+            // Send previous status back based on failure reason.
+            String failureReason = getKeyedString(parameters, FAILURE_REASON, null);
+            if (Strings.isNullOrEmpty(failureReason)) {
```

Comment:
This implies that the poller will be up before the first backup has been scheduled.